src/styles/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  gap: $gap;
+  padding-top: 50px;
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+
+  @each $day, $number in $days {
+    &--start-day-#{$day} {
+      .calendar__day:first-child {
+        margin-left: ($day-size * $number + $gap *$number + $padding);
+      }
+    }
+  }
+
+  @for $date from 1 through 31 {
+    .calendar__day:nth-child(#{$date})::before {
+      content: "#{$date}";
+      margin: auto;
+      font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+      font-size: 30px;
+    }
+  }
+
+  @mixin square ($size) {
+    width: $size;
+    height: $size;
+  }
```

Comment:
Thanks for the heads up. A quick question: Is it really worth using a mixin if I'm only going to use it once? It seems like it would make more sense to use it if I need to repeat a particular piece of code with various values multiple times, rather than just once.